### ISSUE TITLE
Show map stars on the caption panel

### ISSUE THREAD

**Issue by yohanboniface** [umap-project/umap#2282]
And add the "star" button there too ?

The idea is to make the star feature more visible, so people will use it more, and we'll then be able to showcase some "liked" maps.

**Comment by Aurelie-Jallut** [comment]
We could also add the "star" bouton on the search page (and it could be a filter too)

**Comment by Aurelie-Jallut** [comment]
I've just checked By default the Favourites button doesn't appear
![Image](https://github.com/user-attachments/assets/7c1f9e19-852c-418d-8cc4-eafc5ac54b5d)

**Comment by Aurelie-Jallut** [comment]
![Image](https://github.com/user-attachments/assets/c4ed3452-7370-45a5-9aed-6d8d438c29f7)

**Comment by yohanboniface** [comment]
What should we display near the icon when there is no star ?  Nothing ? `0` ?

Seems a bit floating like that, no ?

![Image](https://github.com/user-attachments/assets/66df668f-47af-40a5-9e11-2bc23153c7c8)

**Comment by Aurelie-Jallut** [comment]
Yes, I imagined writing star 0 and displaying ‘Bookmark the map’ on hover.
For greater stability, I'll align the elements on the left, and the info icon on the title :)

In images and on Figma (Legends page - Add favourites) :
![Image](https://github.com/user-attachments/assets/0b913f80-12b5-4bb6-a05a-a168863267da)

I find that Adding the text makes the top of the panel quite heavy : If you want to highlight this action more, we could also do this :

![Image](https://github.com/user-attachments/assets/5c831fc2-f16b-4603-9bf5-20689af1ea2d)

what do you think ?
